src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2858,10 +2858,11 @@ static WellKnownMembers()
                 (byte)(MemberFlags.Method | MemberFlags.Static),                                                                                    // Flags
                 (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.Microsoft_CodeAnalysis_Runtime_Instrumentation - WellKnownType.ExtSentinel),  // DeclaringTypeId
                 0,                                                                                                                                  // Arity
-                    4,                                                                                                                              // Method Signature
+                    5,                                                                                                                              // Method Signature
```

Comment:
Should the instrumentation runtime expose a method to declare it's version or capabilities?